src/load/azext_load/data_plane/utils/constants.py

```diff
@@ -33,3 +33,10 @@ class LoadTestConfigKeys:
 class HighScaleThreshold:
     MAX_ENGINE_INSTANCES_PER_TEST_RUN = 45
     MAX_DURATION_HOURS_PER_TEST_RUN = 3
+
+
+@dataclass
+class LoadCommandsConstants:
+    CONVERT_TO_JMX_CONFIRM_PROMPT = "Once the test is converted, you will not be able to revert. " \
```

Comment:
Shouldnt this be in some resource strings location to localize?

Reply:
We haven't had localization for console messages/logs, will need to setup the `locales` folder. I will create a separate PR for that

Reply:
The CLI team confirmed only English is supported